compiler/src/dotty/tools/dotc/core/Types.scala

```diff
@@ -4633,6 +4633,7 @@ object Types extends TypeUtils {
      */
     def isUnreducibleWild(using Context): Boolean =
       tycon.isLambdaSub && hasWildcardArg && !isMatchAlias
+        && !(args.sizeIs == 1 && defn.isCompiletime_S(tycon.typeSymbol)) // S is a pseudo Match Alias
```

Comment:
`compiletime.ops.S` is intrinsic right? Maybe we could alter symbol / type somehow (in definitions?) to make `isMatchAlias` true?

Reply:
Yeah, there's an argument for that, but I didn't feel like pushing my luck today attempting that..